misc/acrn-config/launch_config/com.py

```diff
@@ -252,9 +240,44 @@ def log_level_set(uos_type, config):
     print('logger_setting="--logger_setting console,level=4;kmsg,level=3;disk,level=5"', file=config)
     print("", file=config)
 
+def function_define(virt_io, vmid, config):
+
```

Comment:
The name of the function is confusing. Better to use another name to show the work done by the function

Reply:
OK, the function tries to create the network, I will use "def tap_network(virt_io, vmid, config)" to instead it.